packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -150,6 +150,12 @@ class PageController extends ScrollController {
   /// direction.
   final double viewportFraction;
 
+  /// This sets a condition if the [PageView] onPageChanged should be called only
+  /// when scrolling reaches the end.
+  ///
+  /// Default vakue is false.
+  bool callPageChangeAtEnd = false;
```

Comment:
This probably shouldn't be public?

Reply:
Makes sense, since we're using that parameter from`PageController`, made it private